server/views/partials/components/case-and-sentences/template.njk

```diff
@@ -0,0 +1,56 @@
+{% from "hmpps/components/court-cases-release-dates/offence-card/macro.njk" import offenceCard %}
+
+{# TODO this code has been lifted from one of the screens in the recall journey, it will eventually evolve into a reusable design component for listing a court case with associated sentences#}
+
+<h2 data-qa="court-case-main-heading" class="govuk-heading-m govuk-!-font-weight-bold govuk-!-margin-top-3 govuk-!-margin-bottom-3">
+  {% if courtCase.reference %}
+    {{ courtCase.reference }} at
+  {% endif %}
+  {# TODO this shouldnt be the formattedOverallConvictionDate - should use the sentencing appearance date - to fix#}
+  {{ courtCase.courtName | default("N/A") }} on {{ courtCase.formattedOverallConvictionDate | default("N/A") }}
+</h2>
+
+{% for sentence in sentences %}
```

Comment:
this whole sentenec card darea is a bit up in the air at the moment. will revisit after tech-analysis and will beef up the tests at that point